<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>942</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Messenger server</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="info">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(225, 213, 255);</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="info_about_users">
       <attribute name="title">
        <string>Info about users</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="1" column="0">
         <widget class="QTableWidget" name="tableWidget">
          <property name="rowCount">
           <number>0</number>
          </property>
          <property name="columnCount">
           <number>5</number>
          </property>
          <column>
           <property name="text">
            <string>Login</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Surname</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Birthdate</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Delete</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="statistics">
       <attribute name="title">
        <string>Statistic info</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QTableWidget" name="tableWidget_2">
          <property name="rowCount">
           <number>0</number>
          </property>
          <property name="columnCount">
           <number>3</number>
          </property>
          <column>
           <property name="text">
            <string>Login</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Last login time</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>IP address</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="server_settings">
       <attribute name="title">
        <string>Server settings</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QWidget" name="database_name" native="true">
          <layout class="QGridLayout" name="gridLayout">
           <property name="leftMargin">
            <number>50</number>
           </property>
           <property name="topMargin">
            <number>35</number>
           </property>
           <property name="rightMargin">
            <number>50</number>
           </property>
           <property name="bottomMargin">
            <number>30</number>
           </property>
           <property name="horizontalSpacing">
            <number>30</number>
           </property>
           <item row="2" column="0">
            <widget class="QLabel" name="label_3">
             <property name="styleSheet">
              <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
             </property>
             <property name="text">
              <string>Port:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="blocked_ip"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_2">
             <property name="styleSheet">
              <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
             </property>
             <property name="text">
              <string>Path to database:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="path_to_file"/>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="port"/>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="db_name"/>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="styleSheet">
              <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
             </property>
             <property name="text">
              <string>Database name:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QPushButton" name="select_btn">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>35</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color:rgb(188, 185, 200);
border-radius:15px;
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
             </property>
             <property name="text">
              <string>Select</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_4">
             <property name="styleSheet">
              <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
             </property>
             <property name="text">
              <string>Blocked IP:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="2">
            <widget class="QPushButton" name="save_btn">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>35</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color:rgb(188, 185, 200);
border-radius:15px;
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
             </property>
             <property name="text">
              <string>Save</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="exit_btn">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>150</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:rgb(188, 185, 200);
border-radius:15px;
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
        </property>
        <property name="text">
         <string>Exit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>942</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
